src/jaxsim/parsers/rod/parser.py

```diff
@@ -326,6 +326,15 @@ def extract_model_data(
 
                 collisions.append(sphere_collision)
 
+            if collision.geometry.mesh is not None:
+                mesh_collision = utils.create_mesh_collision(
+                    collision=collision,
+                    link_description=links_dict[link.name],
+                    method=utils.MeshMappingMethods.VertexExtraction,
+                )
+                if mesh_collision is not None:
+                    collisions.append(mesh_collision)
```

Comment:
Can this ever happen? I guess that in the case `utils.create_mesh_collision` fails, an exception is raised.

Reply:
Unresolving

Reply:
Removed None check